openapi/data_repository_service.openapi.yaml

```diff
@@ -103,8 +103,12 @@ x-tagGroups:
 paths:
   /objects/{object_id}:
     $ref: ./paths/objects@{object_id}.yaml
+  /bulk/objects:
```

Comment:
How about making this endpoint at just `POST /objects` rather than `POST /bulk/objects` ?

Reply:
OK, I'll make that change.

We do need to figure out how to enable multi-part POST here so it can be used with Passports.  this is a TODO